docs/remix_as_code_viewer.md

```diff
@@ -0,0 +1,38 @@
+Remix as code viewer
+=============================
+
+Through Etherscan
+------------------
+
+If there is a contract verified on Etherscan and someone wants to have a look on it in an IDE, mostly for a mutiple part contract verification, Remix provides a quick way to load the contract.
+
+Usually, contract is shown on Etherscan on a URL like this:
+
+`https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7`
+
+In URL, just update `etherscan.io` with `remix.ethereum.org`
```

Comment:
In the URL, update ... or 
In the URL, just update 

(as a note, the use of "just" or "simply" stylistically annoys me when I read it - well it annoys me when the directions say it is simple but I can't do it.  But use it or not - I wouldn't ask you to change that)...actually on second thought- "just" doesn't really bother me.  Simply does. But you didn't use that here.

Reply:
thanks for explanation. I updated it.